guides/common/modules/proc_filter-errata-by-type-severity.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Filter_errata_by_type_or_severity_{context}"]
+= Filter Errata by Type or Severity
+
+Use the following procedure to filter errata by type or severity.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts > All Hosts* and click the name of the host.
+. In the new hosts page click the *Contents* tab.
+. The *Errata* tab of the *Contents* page lists the errata associated with that host.
+. At the top of the page filter by Type by clicking the *Type* button.
+. You can filter to display errata of type Security, Bugfix or Enhancement
+. Filter by severity by clicking the *Severity* button.
+. You can filter to display errata of severity N/A, Low, Moderate, Important or Critical.
+. To unselect your choice, return to the list of options and click the selected option again.
+
+You can use the *Errata* card on the new Host page to pre-filter errata for type before display.
+
+. On the *Errata* card of the new Host page select the type of errata.
+. This takes you to the *Errata* tab of the *Contents* page, where only errata of the selected type are displayed.
```

Comment:
What is this list for? Does it need to be a list? Is it a procedure?

Reply:
@ianf77 - I suggested to you before to go and take some minimalist classes or tutorials with Ingrid Towey. This, for example, is not a step. We do not document UI events. Users have eyes and if we follow minimalist documentation principles, we do not describe the loading and reloading of pages.